thumbv7em-none-eabi.json

```diff
@@ -4,10 +4,15 @@
     "data-layout": "e-m:e-p:32:32-i1:8:32-i8:8:32-i16:16:32-i64:64-v128:64:128-a:0:32-n32-S64",
```

Comment:
I moved these in to the root directory so that they could be found when adding `--target=<triple>` to cargo commands. If you pass a path to the `.json` file, some weird things happen. See rust-lang/rust#24666